org.osgi.test.common/src/main/java/org/osgi/test/common/annotation/ServiceProperty.java

```diff
@@ -30,8 +30,9 @@
 })
 @Retention(RUNTIME)
 @Documented
-public @interface ConfigEntry {
+public @interface ServiceProperty {
```

Comment:
Maybe this is just simply `Property`? As in a property per https://docs.osgi.org/specification/osgi.core/7.0.0/framework.module.html#framework.module.filtersyntax.